pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+# Description
+_provide description here of what the PR is going to do to the code_
+
+# Screenshot(s)
+_add any screenshots that might be useful to whomever is reviewing this PR_
+
+# Notes
+_anything else that doesn't fit in the above categories, but is still important to call out_
+
+# Checklist
+- [ ] My PR has an appropriately descriptive and concise title.
+- [ ] My code follows the Turing Style Guides and best practices.
```

Comment:
I was considering removing this line from the template (at least on the backend, as I do not think we have style guides). Also as it is our capstone and most likely is going to be a resume item, so people looking at it might be unsure what `Turing Style Guides` are? Ignore me, if this is a frontend item.

Reply:
I agree with this, I think moving away from Turing specific language would be a good move, I know that a potential employer probably won't be looking at this section, but it's probably a good habit regardless.

Reply:
Totally fair! I'll chuck it. 🗑️

Reply:
Oh, that said, Turing does have style guides that we were instructed to adhere to in FE. Including links to them here just for your reference. But I agree that it makes sense to ditch 'em now!

JS: https://github.com/turingschool-examples/javascript
HTML: https://github.com/turingschool-examples/html
CSS: https://github.com/turingschool-examples/css

Reply:
These changes look great, thanks for linking the style guides as well!